docs/source/specs/typespec/main.tsp

```diff
@@ -124,7 +124,7 @@ namespace Workspaces {
 
     model CreateWorkspaceRequest extends BasicWorkspace{
         @doc("Parent ID of Workspace A")
```

Comment:
@Ellen-Yi-Dong can you add here note, what does it when parent_id is not state?
something like if parent id in here, it is automatically populated from default workspace.

thanks!